src/librustdoc/html/static/rustdoc.css

```diff
@@ -1773,20 +1785,31 @@ details.rustdoc-toggle > summary.hideme > span {
 }
 
 details.rustdoc-toggle > summary::before {
-	content: "[+]";
 	font-weight: 300;
 	font-size: 0.8em;
 	letter-spacing: 1px;
 	cursor: pointer;
 }
 
+details.rustdoc-toggle > summary::before {
+	content: "[+]";
+}
```

Comment:
Why is this separated out from the existing rule that has the same selector?

Reply:
I tried to merge rules and finally didn't and forgot to put it back as it was. Cleaning it up.